src/target/linux.rs

```diff
@@ -184,7 +184,11 @@ impl LinuxTarget {
         Ok(status)
     }
 
-    fn new(pid: Pid) -> Self {
+    /// Creates a new debug target.
+    /// This method should only be used if you know that `pid` is expecting to be traced
+    /// (e.g., when it called `PTRACE_TRACEME`). Otherwise, you should use `LinuxTarget::attach`
+    /// or `LinuxTarget::launch`.
+    pub fn new(pid: Pid) -> Self {
```

Comment:
Maybe call it new_already_tracing or something like that?

Reply:
Agreed, would be nice to make it more explicit.
How about `LinuxTarget::from_tracee_pid`?

Reply:
That would work too.